dashboard/app/controllers/admin_reports_controller.rb

```diff
@@ -88,6 +88,46 @@ def funometer_by_script_level
     end
   end
 
+  def level_answers
+    authorize! :read, :reports
+
+    @headers = ['Level ID', 'User Email', 'Data']
+    @responses = {}
+    @response_limit = 100
```

Comment:
Hopefully not. At the moment, the intended usage is for levels with low response rates (certain PD levels).